2024-10-17 16:54:04,952 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Configure stats pid to 2007768
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Loading settings from /pbs/home/m/mferey/.config/wandb/settings
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Loading settings from /pbs/home/m/mferey/ML/caverns/lauch/debug_run/wandb/settings
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_init.py:_log_setup():527] Logging user logs to /pbs/home/m/mferey/ML/caverns/lauch/debug_run/wandb/run-20241017_165404-wpktlg4i/logs/debug.log
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_init.py:_log_setup():528] Logging internal logs to /pbs/home/m/mferey/ML/caverns/lauch/debug_run/wandb/run-20241017_165404-wpktlg4i/logs/debug-internal.log
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_init.py:init():567] calling init triggers
2024-10-17 16:54:04,953 INFO    MainThread:2007768 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {}
2024-10-17 16:54:04,954 INFO    MainThread:2007768 [wandb_init.py:init():617] starting backend
2024-10-17 16:54:04,954 INFO    MainThread:2007768 [wandb_init.py:init():621] setting up manager
2024-10-17 16:54:04,963 INFO    MainThread:2007768 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-17 16:54:04,968 INFO    MainThread:2007768 [wandb_init.py:init():629] backend started and connected
2024-10-17 16:54:04,984 INFO    MainThread:2007768 [wandb_init.py:init():721] updated telemetry
2024-10-17 16:54:05,006 INFO    MainThread:2007768 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-10-17 16:54:05,296 ERROR   MainThread:2007768 [wandb_init.py:init():780] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2024-10-17 16:54:05,435 ERROR   MainThread:2007768 [wandb_init.py:init():1205] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/pbs/home/m/mferey/miniconda3/envs/watchmal/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1187, in init
    run = wi.init()
  File "/pbs/home/m/mferey/miniconda3/envs/watchmal/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 786, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2024-10-17 16:54:46,635 INFO    MainThread:2007768 [wandb_settings.py:_apply_login():1941] Applying login settings: {}
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Configure stats pid to 2007768
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Loading settings from /pbs/home/m/mferey/.config/wandb/settings
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Loading settings from /pbs/home/m/mferey/ML/caverns/lauch/debug_run/wandb/settings
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_init.py:_log_setup():527] Logging user logs to /pbs/home/m/mferey/ML/caverns/lauch/debug_run/wandb/run-20241017_165446-g66ddfbx/logs/debug.log
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_init.py:_log_setup():528] Logging internal logs to /pbs/home/m/mferey/ML/caverns/lauch/debug_run/wandb/run-20241017_165446-g66ddfbx/logs/debug-internal.log
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_init.py:init():567] calling init triggers
2024-10-17 16:54:46,657 INFO    MainThread:2007768 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {}
2024-10-17 16:54:46,658 INFO    MainThread:2007768 [wandb_init.py:init():617] starting backend
2024-10-17 16:54:46,658 INFO    MainThread:2007768 [wandb_init.py:init():621] setting up manager
2024-10-17 16:54:46,667 INFO    MainThread:2007768 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-17 16:54:46,668 INFO    MainThread:2007768 [wandb_init.py:init():629] backend started and connected
2024-10-17 16:54:46,683 INFO    MainThread:2007768 [wandb_init.py:init():721] updated telemetry
2024-10-17 16:54:46,703 INFO    MainThread:2007768 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-10-17 16:56:16,728 ERROR   MainThread:2007768 [wandb_init.py:init():780] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-10-17 16:56:16,760 ERROR   MainThread:2007768 [wandb_init.py:init():1205] Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "/pbs/home/m/mferey/miniconda3/envs/watchmal/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1187, in init
    run = wi.init()
  File "/pbs/home/m/mferey/miniconda3/envs/watchmal/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 786, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-10-17 17:01:03,966 WARNING MsgRouterThr:2007768 [router.py:message_loop():77] message_loop has been closed
2024-10-17 17:01:03,966 WARNING MsgRouterThr:2007768 [router.py:message_loop():77] message_loop has been closed
